server.js

```diff
@@ -8,9 +8,9 @@ let rpc = require('./rpc');
 const printer = require('./printer');
 const logger = require('./logger.js');
 const CONFIG = require('./config.js');
+const checkJWT = require('./auth.js').verify;
```

Comment:
since forum controller is the only controller using this for now, can you require this at the top of that.
Also keeps it conditional so this isnt loaded unless a user uses forum or auth